source/core/RefNum.cpp

```diff
@@ -69,7 +69,7 @@ RefNumStorageBase::RefNumHeaderAndData* RefNumStorageBase::CreateRefNumIndex(Ref
 {
     bool isNew;
     UInt32 index = static_cast<UInt32>(IndexFromRefNum(refnum));
-    RefNumHeaderAndData* rnp = CreateRefNumIndexT(index, isNew);
```

Comment:
No, it wasn't a bug, lint recommended changing a non-const reference param to a pointer, so I did.